baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -59,12 +60,26 @@ Additional metadata MAY be appended to values in the form of property set, repre
 Leading and trailing OWS is allowed but MUST be trimmed when converting the header into a data structure.
 
 ## Limits
-1. Maximum number of name-value pairs: `180`.
-2. Maximum number of bytes per a single name-value pair: `4096`.
-3. Maximum total length of all name-value pairs: `8192`.
+1. Maximum number of list-members: `180`.
```

Comment:
Here and in other places, let's use ticks to highlight that this is a variable name from ABNF

```suggestion
1. Maximum number of `list-members`: `180`.
```